.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
 	"files.eol": "\n",
-	"typescript.tsdk": "node_modules\\typescript\\lib"
+	"typescript.tsdk": "node_modules\\typescript\\lib",
```

Comment:
```suggestion
```

VSCode is on TS 3.7 now so we can remove this line again
(it should've never been checked it tbh)